Tests/StoreKitUnitTests/LocalReceiptParserStoreKitTests.swift

```diff
@@ -17,7 +17,7 @@ import Nimble
 import StoreKit
 import XCTest
 
-@available(iOS 14.0, tvOS 14.0, macOS 11.0, watchOS 6.2, *)
+@available(iOS 14.0, tvOS 14.0, macOS 11.0, watchOS 7.0, *)
```

Comment:
oh, [`SKTestSession` is watchOS 7+](https://developer.apple.com/documentation/storekittest/sktestsession#), I'm surprised this even worked before

Reply:
We never compiled this on `watchOS`.